BlockSettleHW/ledger/ledgerDevice.cpp

```diff
@@ -479,9 +479,10 @@ void LedgerCommandThread::processGetPublicKey()
    else {
       logger_->debug(
          "[LedgerCommandThread] getPublicKey - Operation succeeded.\nRoot xpub : "
-         + walletInfo.info_.xpubRoot_ + " \nNested xpub: "
-         + walletInfo.info_.xpubNestedSegwit_ + " \nNativeSegwit: "
-         + walletInfo.info_.xpubNativeSegwit_);
+         + walletInfo.info_.xpubRoot + " \nNested xpub: "
+         + walletInfo.info_.xpubNestedSegwit + " \nNativeSegwit: "
+         + walletInfo.info_.xpubNativeSegwit + "\nLegacy: "
+         + walletInfo.info_.xpubLegacy);
```

Comment:
Using native fmt::format formatting will make it much more clear for the reader, what exactly is printed here. Also, putting a space (' ') before `\n` is pointless.

Reply:
done